pulsar-common/src/main/java/org/apache/pulsar/common/util/netty/EventLoopUtil.java

```diff
@@ -50,7 +50,7 @@
 @Slf4j
 public class EventLoopUtil {
 
-    private static final String ENABLE_IO_URING = "enable.io_uring";
+    private static final String ENABLE_IO_URING = "pulsar.enable.io_uring";
```

Comment:
```suggestion
    private static final String ENABLE_IO_URING = "pulsar.enableUring";
```

Reply:
How about `pulsar.enableIOUring`?

Reply:
@coderzc better than current naming.

Some background: io_uring's lib names as `liburing`, `tokio-rs` has a crate named as `tokio-uring`. Actually, `Uring` is the implementation technology :)

Reply:
`io_uring` is a clear term.

Reply:
Thank you for your answer, `pulsar.enableUring` is good.